### PR TITLE
Implement Laura's suggestion for presentation 01

### PR DESCRIPTION
**Pull request by Felixmil**

### CODE REVIEW

File: 01-introduction.qmd [hunk 1]
```diff
@@ -24,137 +24,185 @@ showtext::showtext_auto()
...
 
-## `{reportingengine}`
+![](resources/01/github.png){fig-align="center"}
 
-::: center-x
-**Automated simulation and reporting workflow**
-:::
+## Documentation Websites
+
+Each package has its own dedicated website containing:
```
Comment (Laura-Villain) [inline 01-introduction.qmd:197]: Maybe add the one page with the links for the doc for each package?

**Review by Laura-Villain — COMMENTED** [review]
I love the changes, I just made one small comment :)

**Comment by Felixmil** [comment]
Done in latest commit

**Review by Laura-Villain — APPROVED** [review]
Perfect :)

**Pull request merged**